### PR TITLE
Seed DB

### PR DESCRIPTION
**Pull request by tochman**
PT Story: https://www.pivotaltracker.com/story/show/129031005

Changes proposed in this pull request:
- adds methods to seeds.rb to create dummy objects
- adds color to terminal prompt
- adds Faker to generate dummy data

What I have learned working on this feature:
- Reminded myself about recursive methods. A recursive function/method is a method that calls itself.
  I used it in my `yes_no` method that is part of the seed-script.
- Also got reminded how messy it can get if you don't extract functionality to methods

**Note: This is just a first version of the seeds file. As we introduce more relationships we will need to update ir.**
#### Usage

just run: 

```
$ rails db:seed
```

Ready for review!

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,115 @@
...
+def add_dish(number, menu, restaurant)
+  price = rand(20..60)
+  prefix = restaurant.cuisine
+  Dish.create(name: "#{prefix} Dish#{number}",
+              price: price,
+              menu: menu,
+              category: Dish::CATEGORIES.sample)
+end
+
+puts '**************************************'.green
```
Comment (diraulo) [inline db/seeds.rb:54]: @tochman you went all fancy on this seeding script huh? 😄

Comment (tochman) [reply db/seeds.rb:54]: Haha

**Comment by tochman** [comment]
Any feedback on this? Imo this script will do for now.

**Comment by diraulo** [comment]
none.. it's okay

**Pull request merged**